ibtk/include/ibtk/HierarchyIntegrator.h

```diff
@@ -420,6 +420,15 @@ class HierarchyIntegrator : public SAMRAI::mesh::StandardTagAndInitStrategy<NDIM
      */
     void registerApplyGradientDetectorCallback(ApplyGradientDetectorCallbackFcnPtr callback, void* ctx = NULL);
 
+    /*!
+     * Perform data initialization after the entire hierarchy has been constructed.
+     *
+     * \note Subclasses should not override the implementation of this function
```

Comment:
Please remove this comment --- the method is not virtual, so cannot be over-ridden in the subclass.